worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +45,66 @@ int main()
 
 	// TODO: implement the rest of the game
```

Comment:
I'll say to you what I've said to 90% of the class: You've. Done. It. So. Remove. The. TODO. Comment!!!